src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/EventHub/NewAzureEventHub.cs

```diff
@@ -36,66 +37,72 @@ public class NewAzureRmEventHub : AzureEventHubsCmdletBase
             Position = 1,
             HelpMessage = "Namespace Name.")]
         [ValidateNotNullOrEmpty]
-        public string NamespaceName { get; set; }
+        [Alias(AliasNamespaceName)]
+        public string Namespace { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             Position = 2,
             HelpMessage = "Namespace Location.")]
         [ValidateNotNullOrEmpty]
+        [ObsoleteAttribute("'Location' property is mark as obsolete and will be depricated in upcoming breaking changes build.", false)]
         public string Location { get; set; }
 
         [Parameter(Mandatory = true,
             ValueFromPipelineByPropertyName = true,
-            Position = 3,
+            Position = 2,
```

Comment:
@v-Ajnava why was this change made? You now have two parameters with the same position, which will cause an error